documentation project blogpost

shannonbux opened the pull request:
wondering if the number of quotes is overkill

shannonbux commented:
Never know who to ask for reviews from. I could use help from anyone, and yet don't want to bother too many people either!

KyleAMathews commented on docs/blog/2018-02-26-documentation-project/index.md:
```diff
@@ -0,0 +1,61 @@
+
+> “We just released our marketing site built with @gatsbyjs. It’s stupid fast, and best of all our marketing folks don’t have to bother us developers about changes anymore :D simplyagree.com.”
+
+And that’s just how much people like Gatsby in general. Now let’s look at how much people like the _docs_:
```
Use "in general" or "generally" three times. You could cut all of them out.

KyleAMathews commented on docs/blog/2018-02-26-documentation-project/index.md:
```diff
@@ -0,0 +1,61 @@
+
+## How much people like the docs :D
+
+In over 60 interviews with Gatsby users, I asked “What do you like about Gatsby?” Documentation comes out as one of the clear winners, as you can see in this list of the top ten things interviewees like about Gatsby, courtesy of Unbird.com analytics:
```
*In my over 60 interviews

KyleAMathews commented on docs/blog/2018-02-26-documentation-project/index.md:
```diff
@@ -0,0 +1,61 @@
+
+Generally, people really like Gatsby docs and Gatsby in general. In fact, we’ve talked about tracking the number of curse words and endearingly pejorative terms that people use when tweeting about Gatsby. Here’s a sampling:
+
+>”HOLY SHIT @gatsbyjs && @NetflifyCMS where have you been all my life”
```
Embed the tweets instead? Makes them more authentic + context. Also people might recognize the tweet authors.

benjaminhoffman replied:
+ always take screenshots of the tweets just in case account gets deleted or tweet deleted.  save the screenshots in a side folder somewhere

KyleAMathews commented on docs/blog/2018-02-26-documentation-project/index.md:
```diff
@@ -0,0 +1,61 @@
+> “For me personally, the most important thing to help me reach my goals with Gatsby is good documentation with examples - which is, in my opinion, already the case compared to other projects.”
+> “Tutorials are very beginner friendly and great for someone who wants to become a better developer.”
+> “The tutorials are very easy to understand and go through great lengths to explain every line of code necessary.”
+
```
There's some great tweets praising the tutorials/docs as well

shannonbux replied:
That's true. Would those be better than these quotes?

jlengstorf replied:
I'd mix both in.

KyleAMathews commented on docs/blog/2018-02-26-documentation-project/index.md:
```diff
@@ -0,0 +1,61 @@
+Many of you have given me valuable feedback during UX research interviews on ways to enhance the tutorials. Here is a brief overview of the issues (see Issue #4175 on Github to follow this and contribute):
+
+Gatsby is beginner-friendly; however, _true_ beginners to programming need an intro to basic tools like the command line, code editors, and browser consoles.
+Upon first visit to Gatsbyjs.org, most visitors click “Get Started.” This is the perfect solution for them if they prefer to jump into coding without reading much. If they prefer step-by-step instructions, it takes some looking around before they see the tutorial tab across the top.
```
Needs spaces between paragraphs?

jlengstorf replied:
This looks like it should be a bulleted list to my eye.

KyleAMathews commented:
Also hard to have too many quotes :-D

benjaminhoffman commented on docs/blog/2018-02-26-documentation-project/index.md:
```diff
@@ -0,0 +1,61 @@
+## What you can do to help
+If you'd like to contribute, look through this list of options and pick one that you'd like to help with! 
+* Go to Issue #4175 to help enhance the tutorials. 
+* If you’d like to contribute to and/or follow other documentation projects, go here to find the [Documentation Project](https://github.com/gatsbyjs/gatsby/projects/3). 
```
same.  will this link always 200?

shannonbux commented on docs/blog/2018-02-26-documentation-project/index.md:
```diff
@@ -0,0 +1,61 @@
+
+After outlining the issues above and attempting to solve them myself, I realized the pooled efforts of the community would be incredibly valuable in creating solutions to these issues. So, I created the Documentation Project in Github to plan and track documentation issues.
+
+The [Documentation Project’s](https://github.com/gatsbyjs/gatsby/projects/3) goal is to make Gatsby documentation clear, organized, and useful. Issues in the project come from the community and/or interviews with community members. This project also provides a centralized way to plan, manage, and track our progress so that we maintain and enhance the docs and tutorials.
```
Good question! This project is new and ought to last for quite a while (i.e. probably the rest of the year at least, if not years), and so this concern would matter down the road. Any recommendations for how to handle that?

jlengstorf replied:
Even if the project is completed, wouldn't GitHub keep a link to it (e.g. closed issues are still available at the same links as when they're open)?

KyleAMathews replied:
it almost certainly would — though "project" is a misnomer here because we'll be using this board forever more. We're using it more like a Trello board.

jlengstorf commented on docs/blog/2018-02-26-documentation-project/index.md:
```diff
@@ -0,0 +1,61 @@
+
+## Enhancing the Tutorials
+
+Just because the tutorials and docs are awesome and quite advanced doesn’t mean they can’t still grow! And we want them to be the absolute _best_ they can be.
```
> Just because the tutorials and docs are awesome and quite advanced...

I always hesitate to use phrasing like this when referring to one's own work. This could be overly cautious, but I prefer to phrase it more like this:

> Just because the docs and tutorials get a lot of praise...

I don't have strong feelings about this — just my own hang-ups where I worry about sounding like a blowhard. 😂

jlengstorf commented on docs/blog/2018-02-26-documentation-project/index.md:
```diff
@@ -0,0 +1,61 @@
+
+Just because the tutorials and docs are awesome and quite advanced doesn’t mean they can’t still grow! And we want them to be the absolute _best_ they can be.
+
+Many of you have given me valuable feedback during UX research interviews on ways to enhance the tutorials. Here is a brief overview of the issues (see Issue #4175 on Github to follow this and contribute):
```
Can we add a link to #4175?

jlengstorf commented on docs/blog/2018-02-26-documentation-project/index.md:
```diff
@@ -0,0 +1,61 @@
+
+## What you can do to help
+If you'd like to contribute, look through this list of options and pick one that you'd like to help with! 
+* Go to Issue #4175 to help enhance the tutorials. 
```
Link.

shannonbux commented:
How to I add images to this PR? I've googled this and the only instructions I can find are how to attach files to comments in the PR, not in the actual directory.

shannonbux commented:
@jlengstorf done with addressing your feedback.

And @KyleAMathews @benjaminhoffman Now I'm seeing patterns with who swears. Mostly white guys haha. But seriously, do the images of curse word tweets feel as under-representative of all gatsby users to you guys as they do to me? If so, we can just use the complimentary tweets about docs.

KyleAMathews commented:
Gloria's isn't the strongest of tweets — it's got several ideas jumbled there.

I'd take out the initial "Generally" — just say "People really like the Gatsby docs" — that's a bit of an overstatement but generally true.

KyleAMathews commented:
>And that’s just how much people like Gatsby. Now, let’s look at how much people like the docs:

This paragraph is adding much and could be removed

KyleAMathews commented:
If you remove that paragraph, the next heading could be "How much do people like the docs? :-D"

shannonbux commented:
@KyleAMathews any idea why the issue mentions aren't automatically linking? The last post I wrote would autofill issue links when I used the # (just like what happens when creating issues and PR's and referencing other issues) but this one isn't doing that. Or maybe I'm imagining things??

KyleAMathews commented:
Issues auto-link here on GitHub but not in Gatsby :-)

KyleAMathews commented on docs/blog/2018-02-26-documentation-project/index.md:
```diff
@@ -39,7 +39,7 @@ So why fix what isn’t broken?
 Many people have contributed to the docs and tutorials already and have done an excellent job. With that being said, there is still room to grow. 
 
-Here is a brief overview of issues that have come up again and again as I've interviewed people and observed them going through the tutorials and docs (see Issue #4175 on Github to follow this and contribute):
+Here is a brief overview of issues that have come up again and again as I've interviewed people and observed them going through the tutorials and docs (see [Issue #4175](/issues/4175/) on Github to follow this and contribute):
```
When you link *within* a domain you can use what's called relative links e.g. if you're on example.com and you click on a link that points to `/about-me/` then your browser will go to `example.com/about-me/`.

If you want to link to *another* domain e.g. from example.com to whataboutme.com — you need to include the full domain for the *other* domain in the link e.g. `whataboutme.com/about-me/`.

So since this blog post will live on gatsbyjs.org and you want to link to github.com — you need to include the full github.com domain in the link or https://github.com/gatsbyjs/gatsby/issues/4175

shannonbux replied:
Ok thank you! I guess I thought since the blog resides in github, then they shared the same domain. So it's more about the domain where the viewer views them, not where the files are being stored?

KyleAMathews replied:
Yeah -- the files here are raw materials that Gatsby uses to build the site. So we need to prep materials here for what's needed on the real site. The blog isn't here. Just raw materials for the blog.

shannonbux commented:
@jlengstorf since I already addressed your feedback, is there a way to show that? Should I click dismiss review?

KyleAMathews commented:
@shannonbux looks good to me!

Anything else you'd like to do?

shannonbux commented:
@KyleAMathews nope this is ready! Looks like I can merge it??

shannonbux commented:
Wohoo! That felt great! 💯

KyleAMathews commented:
Hehe, great work! Enjoy the feelings of complete raw power soaking into your soul :-D
